core/src/main/scripts/importer/validateData.py

```diff
@@ -5072,8 +4915,7 @@ def main_validate(args):
                                          hugo_entrez_map=None,
                                          alias_entrez_map=None,
                                          gene_set_list=None,
-                                         gene_panel_list=None,
-                                         treatment_map=None)
```

Comment:
Are you aware that you have eliminated the existing mechanism that checks for and acts upon entities with the same name being present in the database! How is this currently handled for Generic Assay entities? When this is not handled in the validator, the user is not informed of the consequences of import of the current dataset.

Reply:
Same to the previous comment.